### PR TITLE
Add python-slugify recipe

### PR DESCRIPTION
**Pull request by proinsias**
- [ ] Tidy up `meta.yaml` when build succeeds.

### CODE REVIEW

File: recipes/python-slugify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipes/python-slugify/meta.yaml]: Please add `conda-build` [`entry_points`]( http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points ).

```yaml
  entry_points:
    - slugify = slugify.slugify:main
```

**Comment by jakirkham** [comment]
One comment, which should fix the build failure too.

File: recipes/python-slugify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#  entry_points:
+#    - slugify = slugify:main
```
Comment (jakirkham) [inline recipes/python-slugify/meta.yaml]: Why commented? Also think it is `slugify.slugify:main`.

Comment (proinsias) [inline recipes/python-slugify/meta.yaml]: The build failed with both `slugify.slugify:main` and `slugify:main`.
Any ideas?

File: recipes/python-slugify/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/python-slugify/meta.yaml]: Done below.

**Comment by jakirkham** [comment]
Thanks @proinsias.

**Pull request merged**